### PR TITLE
Add links to collections

### PR DESCRIPTION
**Pull request by edpaget**
Add a 'Collect' link in the header that links to an index of all the public collections. A logged in user can also view their current collections from the same place or from the account dropdown. Individual collections are linked at `collections/user-slug/collection-slug`.

### CODE REVIEW

File: app/collections/show.cjsx [hunk 1]
```diff
@@ -11,43 +11,41 @@ module?.exports = React.createClass
...
+      @setState {collection: collection, page: page}, =>
+        @subjectsRequest().then (subjects) => @setState subjects: subjects
 
   goToPage: (n) ->
     @transitionTo(@props.path, @props.params, {page: n})
     @setData(n)
 
   subject: (d, i) ->
-    <img key={d.id} src={getSubjectLocation(d).src} />
+    <SubjectViewer subject=d />
```
Comment (aweiksnar) [inline app/collections/show.cjsx]: Maybe worth wrapping in curleys

**Comment by aweiksnar** [comment]
Looks really great. Only 2 things I noticed are:
  - `owner-card.styl` should become `owned-card.styl` for the css to compile
  - Since project-card/project-card-list are becoming generalizable as owned-card/owned-card-list, it may be nice to declare propTypes for a sort of mini documentation

**Comment by edpaget** [comment]
Sure I'm unfamiliar with propTypes is there a good place to look for them?

**Comment by aweiksnar** [comment]
Yeah here's a link https://facebook.github.io/react/docs/reusable-components.html

**Comment by edpaget** [comment]
Thanks got them added.

**Comment by aweiksnar** [comment]
Sweet thanks a ton!

**Pull request merged**